themes/SuiteP/include/DetailView/tab_panel_content.tpl

```diff
@@ -54,13 +56,19 @@
                 {{counter name="fieldCount" start=0 print=false assign="fieldCount"}}
                 {{foreach name=fieldIteration from=$colData key=field item=subField}}
 
+                    {{if !isset($subField.name) || !$subField.name}}
```

Comment:
You are still hiding the field, apply same fix to DetailView tpl from EditView tpl.